tools/jit/gen_jit_dispatch.py

```diff
@@ -238,6 +238,10 @@ def is_out_variant(decl):
     return decl['name'].endswith('_out')
 
 
+def is_backward_op(decl):
+    return decl['name'].endswith('_backward') or decl['name'].endswith('_backward_out')
```

Comment:
This looks a bit hacky but there is an example above searching for "_out" and another example searching for "backward": https://github.com/pytorch/pytorch/blob/master/tools/autograd/gen_variable_type.py#L574

Reply:
can we use the same set of filters as in python bindings? (and add a comment about it)

Reply:
> can we use the same set of filters as in python bindings? (and add a comment about it)

Do you think I should use the entire SKIP_PYTHON_BINDINGS / even directly references it?

It has a lot other stuff like '.*_forward', 'index', .. - I haven't verified but do you think these can all be safely deleted from JIT binding?